src/app/home-page/widget/entry-box/entry-box.component.html

```diff
@@ -43,22 +43,26 @@
     <mat-card-content class="p-3 h-100">
       <div *ngIf="totalEntries !== 0 || isLoading; else noEntries">
         <mat-form-field appearance="outline" class="search-form-field w-100 pb-0">
-          <input
-            matInput
-            data-cy="search-field"
-            [(ngModel)]="filterText"
-            (keyup)="onTextChange($event)"
-            placeholder="Search your {{ entryTypeLowerCase }}s..."
-          />
+          <mat-label>Search your {{ entryTypeLowerCase | titlecase }}s...</mat-label>
+          <input matInput data-cy="search-field" [(ngModel)]="filterText" (keyup)="onTextChange($event)" [matAutocomplete]="auto" />
+          <button *ngIf="filterText" mat-button matSuffix mat-icon-button (click)="clearSearch()" type="button">
+            <mat-icon>cancel</mat-icon>
+          </button>
         </mat-form-field>
+        <mat-autocomplete #auto="matAutocomplete" (optionSelected)="navigateToEntry($event.option.value)">
+          <mat-option *ngFor="let result of listOfResults" [value]="result.path" class="no-wrap">{{ result.prettyPath }}</mat-option>
+          <div *ngIf="totalResults > 5" class="date-display no-wrap p-2 px-4">
+            Showing 5 of {{ totalResults }} matching {{ entryTypeLowerCase }}s
```

Comment:
Can the 5 here be a variable shared with `this.listOfResults = myEntries.body.slice(0, 5);` (different file) below? 
Seems like the kind of thing where someone may change this and forget to change the other one